other-builds/ndkbuild/native-audio/app/build.gradle

```diff
@@ -0,0 +1,41 @@
+apply plugin: 'com.android.application'
+// pointing to cmake's source code for the same project
+def REMOTE_SRC_ROOT = '../../../../' + rootProject.getName() + '/' +
+        project.getName() + '/src/main'
+
+android {
+     compileSdkVersion 23
+     buildToolsVersion '23.0.2'
+
+     defaultConfig {
+        applicationId 'com.example.nativeaudio'
+        minSdkVersion 9
+        targetSdkVersion 23
+        ndkConfig.abiFilters 'x86', 'x86_64', 'armeabi', 'armeabi-v7a', 'arm64-v8a'
+    }
+    sourceSets {
+        main {
+            manifest.srcFile "${REMOTE_SRC_ROOT}/AndroidManifest.xml"
+            java.srcDirs = ["${REMOTE_SRC_ROOT}/java"]
+            res.srcDirs = ["${REMOTE_SRC_ROOT}/res"]
+            assets.srcDirs = ["${REMOTE_SRC_ROOT}/assets"]
+        }
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled = false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'),
+                          'proguard-rules.pro'
+        }
+    }
+    externalNativeBuild {
+        ndkBuild {
+            path 'android.mk'
```

Comment:
Oh, thank you!  my mac by default is case-insensitive so this slipped away: I will fix it right now